src/controllers/facilitiesController.js

```diff
@@ -192,5 +194,30 @@ class FacilitiesController {
       return Response.errorResponse(res, 500, res.__('server error'));
     }
   }
+
+  /**
+  * @description rate facility method
+  * @static
+  * @param {Object} req
+  * @param {Object} res
+  * @returns {Object} Facility with ratings
+  * @memberof FacilitiesController
+  */
+  static async rateFacility(req, res) {
+    const { user } = req;
+    const { facilityId } = req.params;
+    const { rating } = req.query;
+    try {
+      const result = await FacilityService.rateFacility(facilityId, user, rating);
+      if (result === stringHelper.facilityNotFound) {
```

Comment:
couldn't these two `If statements` be merged into one

Reply:
they represent different scenarios, the first one is when a facility isn't found and the second if the facility is there but the user hasn't visited it yet

Reply:
but the statements inside are the same